tests/sentinel/tests/14-debug-command.tcl

```diff
@@ -0,0 +1,9 @@
+source "../tests/includes/init-tests.tcl"
+
+test "sentinel debug test with arguments and without argument" {
+   set current_info_period [lindex [S 0 SENTINEL DEBUG] 1]
+   S 0 SENTINEL DEBUG info-period 8888
+   assert { [lindex [S 0 SENTINEL DEBUG] 1] == {8888} }
```

Comment:
Maybe use instead:
```
assert_equal [string first [S 0 SENTINEL DEBUG] 8888] -1
```
This way it won't break if tomorrow `info-period` will be at a diff location. 
And better use more accurate kind of assert for the test.

Reply:
Addressd, please take a look. Thanks

Reply:
I update to previous way: use lindex, please take a look, Thanks